src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -924,6 +969,13 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         // The scroll bar's ValueChanged handler will actually move the viewport
         //      for us.
         _scrollBar.Value(static_cast<int>(newValue));
+
+        if (_terminal->IsSelectionActive() && pointerPoint.Properties().IsLeftButtonPressed())
+        {
+            // If user is mouse selecting and scrolls, he then points at new character.

```

Comment:
```suggestion
            // If user is mouse selecting and scrolls, they then point at new character.
```

Reply:
Is this how do you write that? Well, understand the sad purpose of this, but it doesn't seem grammatically correct.

Reply:
I agree it sounds weird; guess I just got used to it. It's one of the ways I learned it and it's generally pretty easy to incorporate.